### PR TITLE
Merge and cleanup #13143

### PR DESCRIPTION
**Pull request by chancancode**

### CODE REVIEW

**Comment by chadhietala** [comment]
LGTM

File: packages/ember-glimmer/tests/integration/content-test.js [hunk 1]
```diff
@@ -67,24 +67,49 @@ class DynamicContentTest extends RenderingTest {
...
     this.assertInvariants();
 
     this.runTask(() => set(this.context, 'message', 'hello'));
 
-    this.assertText('hello');
+    this.assertContent('hello');
+    this.assertInvariants();
+  }
+
+  ['@test it can render a capitalized path with no deprication']() {
```
Comment (GavinJoyce) [inline packages/ember-glimmer/tests/integration/content-test.js]: `... with no deprecation `

File: packages/ember-glimmer/tests/integration/content-test.js [hunk 2]
```diff
@@ -67,24 +67,49 @@ class DynamicContentTest extends RenderingTest {
...
+    expectNoDeprecation();
+
+    this.renderPath('CaptializedPath', { CaptializedPath: 'noDeprication' });
```
Comment (GavinJoyce) [inline packages/ember-glimmer/tests/integration/content-test.js]: `noDeprecation`

**Comment by chancancode** [comment]
@GavinJoyce saves the day :smile:

**Comment by chancancode** [comment]
Uh, I fixed it on the branch but github ended up merging the stale HEAD (presumably due to the outage). Fixed it again on master.

**Comment by chancancode** [comment]
Merged in 4689127e4b65c37f803f6fd40b27a58bcfdc3330 and bdb81c40f4ca21b438e3ae39f6056eea13b43f3f

**Pull request closed without merging**